src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -417,7 +417,7 @@ object Initializer extends SparkSessionWrapper {
     logger.log(Level.INFO, "Initializing Config")
     val config = new Config()
     val orgId = if (dbutils.notebook.getContext.tags("orgId") == "0") {
-      dbutils.notebook.getContext.tags("browserHostName").split("\\.")(0)
+      dbutils.notebook.getContext.apiUrl.get.split("\\.")(0).split("/").last
```

Comment:
we can get orgId & Cloud from the Spark tags

Reply:
Right, I'm just not entirely sure which of them get populated from within the different job types (i.e. notebook vs spark submit vs call to Main class). I will do some testing. Opening [issue](https://github.com/databrickslabs/overwatch/issues/115) to track this and will likely need to do even more due diligence for E2.

`spark.databricks.clusterUsageTags.clusterOwnerOrgId`
`spark.databricks.clusterUsageTags.cloudProvider`
`spark.databricks.clusterUsageTags.clusterAllTags`